src/components/error-message/index.md.njk

```diff
@@ -43,6 +43,12 @@ For each error:
 - use a red border to visually connect the message and the question it belongs to
 - if the error relates to specific text fields within the question, give these a red border as well
 
+To help users of some assistive technologies, like screen readers, the error message component includes a hidden 'Error:' prefix before the message.
```

Comment:
I think it'd be good to be explicit about the behaviour for people who don't know what it means.

```suggestion
To help users of some assistive technologies, like screen readers, the error message component includes a hidden prefix before the message. 

This gives more context to users who cannot see the message, by reading out the word "error" before the message.   
```